tools/ci_build/build.py

```diff
@@ -368,9 +368,13 @@ def convert_arg_line_to_args(self, arg_line):
     )
     parser.add_argument("--gdk_platform", default="Scarlett", help="Sets the GDK target platform.")
 
+    # Note: This arg is for specifying if this build includes macosx platform support in the apple xcframework.
+    # Different from helper function is_macOS() that's for verifying system platform info that we are building on.(i.e. windows/linux/macos)
+    parser.add_argument("--macosx", action="store_true", help="build for macosx")
```

Comment:
it's not so obvious when `--macosx` should be used with the current help text.
do we need to add another option or could it be inferred by `args.build_apple_framework and not args.ios`?